core/src/components/tab-bar/test/basic/tab-bar.e2e.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('tab-bar: basic'), () => {
+    test('should not have visual regressions', async ({ page }) => {
+      await page.setContent(
```

Comment:
We already have test coverage for this here: https://github.com/ionic-team/ionic-framework/blob/068d0038602485085face4a443e473bf1d5cc227/core/src/components/tab-button/test/basic/tab-button.e2e.ts#L6
Maybe the test needs to move though since it's not obvious?

Reply:
I guess it's confusing because in Segment we keep the tests on `ion-segment` and have none in `ion-segment-button` but in Radio we have tests in both `ion-radio-group` and `ion-radio`. I could remove mine though, up to you.

Reply:
I think we should remove the test here since it is duplicating test effort. However, I do agree that where some of these tests live are confusing. Maybe we can consolidate in a separate PR?